src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
+
+function convertToNumber(string) {
+  const salaryString = string;
+  const salaryNumber = salaryString.replace('$', '').replace(',', '');
+
+  return +salaryNumber;
+}
+
+function sortBySalary(items, list) {
+  items.sort(
+    (a, b) =>
+      convertToNumber(b.dataset.salary) - convertToNumber(a.dataset.salary),
+  );
+
+  list.innerHTML = '';
+  items.forEach((item) => list.appendChild(item));
+}
```

Comment:
This function does two things: it sorts the items and then it modifies the DOM. Consider splitting it into two separate functions. One function can sort the items and return the sorted list. The other function can take a list of items as an argument and update the DOM. This way, each function has a single responsibility which makes your code more maintainable and testable.